### PR TITLE
travis: run the tests in the GCE trusty machine

### PR DESCRIPTION
**Pull request by elopio**
We applied a hack in travis to get a xenial LXC and run the tests in
there. Something changed and now the tests are taking an excessive
amount of time, and timing out. We no longer need this tests to run in
xenial because we are already running them in Jenkins, so we can get rid
of the hack.

LP: #1555386

### CODE REVIEW

**Comment by sergiusens** [comment]
Instead of `/usr/bin/flake8` do `python3 -m flake8`

File: .travis.yml [hunk 1]
```diff
@@ -14,30 +14,13 @@ env:
   - TEST_SUITE=static
   - TEST_SUITE=unit
   - TEST_SUITE=integration
```
Comment (sergiusens) [inline .travis.yml]: shouldn't integration go away as well?

**Comment by elopio** [comment]
good bye lxc, I'll miss you. :wave:

**Comment by sergiusens** [comment]
:+1: 
Please rebase, once tests pass feel free to merge

**Comment by kyrofa** [comment]
:+1: looks good!

**Pull request merged**